src/test/java/ruslan/shastkiv/bookstore/utils/OrderTestUtils.java

```diff
@@ -34,80 +28,219 @@ public class OrderTestUtils {
     public static final Long NOT_EXISTED_ORDER_ITEM_ID = 100L;
 
+    public static final LocalDateTime ORDER_DATE
+            = LocalDateTime.of(2024, 12, 6, 15, 30, 0, 0);
+
+    /**
+     * Creates a {@link PlaceOrderRequestDto} with a formatted shipping address for the specified
+     * user ID. The shipping address is generated using a custom address format and the user ID.
+     * This method is typically used when creating a request to place an order, where the shipping
+     * address depends on the user.
+     *
+     * @param userId the ID of the user for whom the shipping address is created.
+     * @return a {@link PlaceOrderRequestDto} with the generated shipping address.
+     */
+
     public static PlaceOrderRequestDto createPlaceOrderRequestDto(Long userId) {
         return new PlaceOrderRequestDto(
                 CUSTOM_SHIPPING_ADDRESS.formatted(userId)
         );
     }
 
-    public static OrderItemDto createOrderItemDto(Long id) {
-        return new OrderItemDto(
-                id,
-                id,
-                id.intValue()
-        );
-    }
+    /**
+     * Creates an {@link OrderDto} with the specified order ID, a list of {@link OrderItemDto}s,
+     * and the order status. The order date is set to the current time, and the total is set
+     * to a fixed value (9). This method is useful for creating an {@link OrderDto} with
+     * predefined values for testing or other purposes.
+     *
+     * @param id     the ID of the order;
+     * @param items  the list of {@link OrderItemDto}s representing the items in the order;
+     * @param status the {@link Order.Status} representing the current status of the order;
+     * @return an {@link OrderDto} instance with the specified values.
+     */
 
     public static OrderDto createOrderDto(Long id, List<OrderItemDto> items, Order.Status status) {
         return new OrderDto(
                 id,
                 id,
                 items,
                 LocalDateTime.now(),
-                BigDecimal.valueOf(9L),
+                items.stream()
+                        .map(orderItem -> new BigDecimal(orderItem.quantity())
+                                .multiply(new BigDecimal(orderItem.id())))
+                        .reduce(BigDecimal.ZERO, BigDecimal::add),
                 status);
     }
 
-    public static List<OrderDto> getOrderDtosFromMvcResult(
-            MvcResult result,
-            ObjectMapper objectMapper) throws Exception {
-        return objectMapper.convertValue(
-                objectMapper.readTree(result.getResponse().getContentAsString())
-                        .get("content"), new TypeReference<List<OrderDto>>() {}
-        );
+    /**
+     * Creates an instance of {@link UpdateOrderStatusRequestDto} with the specified order status.
+     *
+     * @param status the new status to set for the order;
+     * @return an {@link UpdateOrderStatusRequestDto} containing the specified status.
+     */
+
```

Comment:
Do you really need all of these javadoc comments? As a best practice currently there is the next approach:
if you write your own API (not expose it to the other devs/companies to use, meaning your API isn't public) it's better to write the code in such a way there is no need to comment it. Excessive comments could be hard to maintain and require extra time to write it

Reply:
if you do decide to have javadoc, then please remove extra empty line after comment (before method declaration)